### PR TITLE
Add MUI icon (in the scope of renaming from Material-UI)

### PR DESCRIPTION
**Pull request by SimonGolms**
mui

**Issue:** #6726 
**Alexa rank:** [14537](https://www.alexa.com/siteinfo/mui.com)

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### Description
- I used the [SvgMuiLogo.tsx](https://github.com/mui-org/material-ui/blob/master/docs/src/icons/SvgMuiLogo.tsx) from GitHub repo, which is also used on the [website](https://mui.com/).
- I took the color `#007FFF` directly from the svg icon
- Sketch could not display the path correctly, so I used Figma to adjust the icon on a viewbox of `0 0 24 24`.
- With the SVGO command line tool I optimized the icon with a precision of 5.
- For completeness I have added `Material-UI` as aka

As recommended in the issue #6726, I would do an extra PR to delete the old icon as soon as this icon was added and then also remove the aka again.

### CODE REVIEW

File: _data/simple-icons.json [hunk 1]
```diff
@@ -6186,6 +6186,16 @@
...
         },
+        {
+            "title": "MUI",
+            "aliases": {
+                "aka": [
+                    "Material-UI"
+                ]
+            },
+            "hex": "007FFF",
+            "source": "https://github.com/mui-org/material-ui/blob/master/docs/src/icons/SvgMuiLogo.tsx"
```
Comment (service-paradis) [inline _data/simple-icons.json:6197]: Can you use the permalink here?
https://github.com/mui-org/material-ui/blob/353cecb5391571163eb6bd8cbf36d2dd299aaf56/docs/src/icons/SvgMuiLogo.tsx

File: icons/mui.svg [hunk 2]
```diff
@@ -0,0 +1 @@
...656.66656 0 0 0-.3322-.57935.66657.66657 0 0 0-.6678-.00066L8.91225 9.79864a.66675.66675 0 0 1-.66535 0L4.44346 7.60857a.66668.66668 0 0 0-.99936.57735v6.26688a.66664.66664 0 0 1-.33182.5769.66612.66612 0 0 1-.33245.0902.6673.6673 0 0 1-.33309-.0878L.33601 13.8254a.66652.66652 0 0 1-.2462-.2447A.66682.66682 0 0 1 0 13.2454L.01866 2.29176a.6667.6667 0 0 1 .99869-.57668l7.2302 4.15278a.6667.6667 0 0 0 .66402 0l7.22822-4.15278a.66644.66644 0 0 1 .3333-.08855.6663.6663 0 0 1 .3329.08984.66712.66712 0 0 1 .2436.24419.66717.66717 0 0 1 .0889.3332V13.2501c0 .1172-.0308.2323-.0895.3338a.66671.66671 0 0 1-.2445.2442l-3.7855 2.18a.66709.66709 0 0 0-.2446.2448.66606.66606 0 0 0-.0891.3344.6671.6671 0 0 0 .0906.334.66608.66608 0 0 0 .2458.2435l2.0881 1.1887c.1009.0575.215.0876.3311.0874a.66703.66703 0 0 0 .3309-.0887l4.4568-2.5594Zm.438-9.82827a.66603.66603 0 0 0 .0909.3355.66628.66628 0 0 0 .2473.24416.66676.66676 0 0 0 .6711-.00831l2.0001-1.20004a.6662.6662 0 0 0 .2372-.243A.66658.66658 0 0 0 24 4.7645V2.31909a.6673.6673 0 0 0-.0911-.33526.66692.66692 0 0 0-.584-.33017.66692.66692 0 0 0-.3343.09475l-2 1.20004a.66646.66646 0 0 0-.2373.243.66755.66755 0 0 0-.0867.32834V5.9652v-.00067Z"/></svg>
```
Comment (service-paradis) [inline icons/mui.svg:1]: The icon is a perfect match! I'm just wondering if the precision of 5 was necessary?

**Review by service-paradis — CHANGES_REQUESTED** [review]
Thank you @SimonGolms for your contribution! A really good work as usual!

I made 2 comments. Can you take a look at them? There will be a major release soon and it would be great if we can include this change!

File: _data/simple-icons.json (hunk 1 above)
Comment (SimonGolms) [reply _data/simple-icons.json]: Thanks for the correction - I updated the source link

File: icons/mui.svg (hunk 2 above)
Comment (SimonGolms) [reply icons/mui.svg]: True, looks also good with a precision of 3 in my opinion :)

File: _data/simple-icons.json (hunk 1 above)
Comment (service-paradis) [reply _data/simple-icons.json]: It looks like you commited Jest Cache folder. Can you remove it?

Comment (SimonGolms) [reply _data/simple-icons.json]: Removed

**Pull request merged**